### ISSUE TITLE
Algolia search in the documentation for Fumadocs returns 403

### ISSUE THREAD

**Issue by MacoMoon** [fuma-nama/fumadocs#681]
### To Reproduce

1. Go to https://fumadocs.vercel.app/docs/headless
2. Open network tab
3. Click Search to open the Search dialog
4. Attempt to search for any content, e.g. "Tag Filter"

### Current vs. Expected behavior

*Expected behavior:*
Search results appear.

*Current behaviour:*
Network error 403 Forbidden.
{"message":"This operation cannot be processed, the application is blocked. Contact us to unblock it.","status":403}

### Provide environment information

```bash
Chrome Version 127.0.6533.89 (Official Build) (arm64)
```

### Which area(s) are affected? (Select all that apply)

Examples

**Comment by fuma-nama** [comment]
Duplicate of https://github.com/fuma-nama/fumadocs/issues/677
